src/main/java/com/Model/Event.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Author: Zhe
+ * Date: May30-2020
+ * Event class is going to store the details of patient attended events
+ */
+package com.Model;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+public class Event {
+
+    @Id
+    @GeneratedValue (strategy = GenerationType.SEQUENCE)
+    private long Id;
+    private Date date;
+    private String name;
+
+    @OneToOne
+    private Place place;
+
+    @ManyToMany
+    private List<Patient> attendedPatients;
+
+    @Autowired
+    public Event(){}
```

Comment:
I think the default constructor should initialize all variables to some default value, and do so by invoking the other constructor with the default values. Right now if someone calls the default constructor you will end up with a lot of null values, and an empty list of patients, which is probably not what you want. It is not always obvious however what the default values should be.